### ISSUE TITLE
Invalid Attribute error

### ISSUE THREAD

**Issue by mcfearsome** [hashicorp/terraform-k8s#20]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### terraform-k8s & Kubernetes Version

operator: 0.1.1-alpha
k8s: `Server Version: version.Info{Major:"1", Minor:"13+", GitVersion:"v1.13.12-eks-eb1860", GitCommit:"eb1860579253bb5bf83a5a03eb0330307ae26d18", GitTreeState:"clean", BuildDate:"2019-12-23T08:58:45Z", GoVersion:"go1.11.13", Compiler:"gc", Platform:"linux/amd64"}`

### Affected Resource(s)

workspace

### Terraform Configuration Files

```hcl
terraform {
  backend "remote" {
    organization = "workarea-commerce"
    workspaces
    {
      name = "egdod-staging-tf-egdod-staging-helios"
    }
  }
}
variable "cluster_name" {}
variable "recaptcha_api_key" {}
variable "recaptcha_secret" {}

module "operator" {
  source = "app.terraform.io/workarea-commerce/helios/aws"
  version = "1.0.6"
  cluster_name = var.cluster_name
  recaptcha_api_key = var.recaptcha_api_key
  recaptcha_secret = var.recaptcha_secret
}
```

### Debug Output

https://gist.github.com/mcfearsome/21990e1580f3413f8eec954ad61c4f16

### Expected Behavior

I expect the workspace to get planned/applied

### Actual Behavior

Throwing `Invalid Attribute` error. 

This run was only a version bump, with no changes to the inputs or their values.

### Steps to Reproduce

This is happening intermittently, I do not have a concrete set of steps. Sometimes it will throw this and I will still see the plan/apply in TF cloud, others it just stops and never tries again.

It would be helpful if the error output included what attribute it is having trouble with at the very least. It also seems like the status on the workspace doesn't get properly updated when this happens as it is currently `applied` in my cluster.

### Important Factoids

I have customized the helm chart to allow for workspace resources to be in any namespace

**Comment by mcfearsome** [comment]
restarting the operator triggered the run to complete successfully with no other changes

**Comment by mcfearsome** [comment]
I have also gotten it to work by making meaningless changes to the workspace to trigger a new run. Mainly I have just been changing the order of the variables array. At most 2 additional attempts have been needed so far.

**Comment by joatmon08** [comment]
For tracking, may be related to issue with go-tfe. hashicorp/go-tfe/issues/116

**Comment by mcfearsome** [comment]
So I recently started experiencing this issue regularly, to the point that the operator is basically unusable. After reading the linked issue and poking around a bit, I was able to get the operator to start working by adding in a `time.Sleep(5 * time.Second)` after the configuration file was uploaded and before the run is triggered in `CreateRun`. Not necessarily an ideal solution but it has made it more stable for me.
